Makefile

```diff
@@ -318,9 +318,18 @@ config_updater_vault_secret:
 		--context=$(cluster) \
 		--confirm \
 		--kubeconfig=/_kubeconfig
-	mkdir -p $(build_farm_credentials_folder)
-	oc --context "$(cluster)" sa create-kubeconfig -n ci config-updater > "$(build_farm_credentials_folder)/sa.config-updater.$(cluster).config"
-	make dry_run=false ci-secret-generator
+
+	./clusters/psi/create_kubeconfig.sh "$(build_farm_credentials_folder)/sa.config-updater.${cluster}.config" ${cluster} $@ ci ${API_SERVER_URL} config-updater-token-version-$(token_version)
+
+	echo "We have following secrets:"
+	ls $(build_farm_credentials_folder)
```

Comment:
```suggestion
```

NIT: I would remove those 2 lines.

Reply:
I guess its better to show which secrets are included.